_includes/snippets_library/learnhpc_slurm/_config_options.yml

```diff
@@ -0,0 +1,67 @@
+#------------------------------------------------------------
+# ComputeCanada: Graham + SLURM
```

Comment:
Please rephrase the title.

Reply:
Thanks for catching that, I'll edit this whole comment now!